packages/components/src/FilterBar/context/reducer/setupFilterBarState.ts

```diff
@@ -1,41 +1,43 @@
 import { Filters, FiltersValues } from "../../types"
 import { FilterBarState } from "../types"
+import { updateDependentFilters } from "../utils/updateDependentFilters"
 
 export const setupFilterBarState = <ValuesMap extends FiltersValues>(
-  filters: Filters<ValuesMap>
+  filters: Filters<ValuesMap>,
+  values: Partial<ValuesMap>
 ): FilterBarState<ValuesMap> => {
   const state = filters.reduce<FilterBarState<ValuesMap>>(
     (baseState, { id, name, isRemovable, isUsableWhen }) => {
       const hasDependency = isUsableWhen !== undefined
+      const isUsable = true
```

Comment:
Seems a bit odd to make this a variable when it has no logic. Could it potentially be controllled from outside this setup and come from `values` with a default to `true`?

Reply:
Ah yes, I can probably clean this one up. It used to be `!hasDependency` but it didn't work lol.